base/linalg/triangular.jl

```diff
@@ -1900,14 +1907,10 @@ end
 function sqrtm{T}(A::UnitUpperTriangular{T})
```

Comment:
It seems like we could replace this with
```jl
sqrtm{T}(A::UnitUpperTriangular{T}) = sqrtm(A, Val{true})
```
in terms of the abovementioned method.  Given a `Val` method that is type-stable, the only remaining reason to duplicate the code here seems to be to save the `sqrt` call for the diagonal elements, but I doubt this is worth the trouble since there are only O(n) such calls.